lib/galaxy/tools/deps/views.py

```diff
@@ -45,6 +45,11 @@ def manager_dependency(self, **kwds):
     def resolver_dependency(self, index, **kwds):
         return self._dependency(**kwds)
 
+    def install_dependencies(self, index, requirements):
+        is_installed = self._dependency_manager.install_all(requirements)
```

Comment:
Hmm, this would prefer resolvers that can do `install_all` ... I guess going through `resolve_all` would be better.